src/local_options

```diff
@@ -46,6 +46,7 @@
 #undef USE_ICONV
 #undef IPV6
 #undef DTRACE
+#define POST_CREATE
```

Comment:
Please do not create new defines, instead , create an runtime config .  see base/internal/rc.cc

Also, in order to get merged, this must remian off by default Otherwise we may break existing code, which may include a "post_create" function before. (unlikely, but still)